src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -800,54 +800,14 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
 void Transformations::runLptPasses(const std::vector<ov::element::Type>& defaultPrecisions) {
     using namespace ov::pass::low_precision;
     ov::pass::Manager lptManager("CPU:LPT");
+
 #if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+    auto quantizationRestrictions = std::vector<QuantizationGranularityRestriction>(
```

Comment:
Please double check `quantizationRestrictions` for both platforms: previously, it was empty for ARM and had `Convolution` and `ConvolutionBackpropData` restrictions for X64. Now it is vice versa.

Reply:
You're right, thanks, fixed!

Reply:
Discussed offline: it was agreed to update the current plugin tests to cover these restrictions

Reply:
Added the test